hypervisor/arch/x86/guest/vlapic.c

```diff
@@ -2109,7 +2109,7 @@ int32_t vlapic_x2apic_write(struct acrn_vcpu *vcpu, uint32_t msr, uint64_t val)
 	 */
 	vlapic = vcpu_vlapic(vcpu);
 	if (is_x2apic_enabled(vlapic)) {
-		if (is_lapic_pt(vcpu->vm)) {
+		if (is_lapic_pt_configured(vcpu->vm)) {
 			switch (msr) {
 			case MSR_IA32_EXT_APIC_ICR:
```

Comment:
Use is_lapic_pt_enabled here and remove is_lapic_pt_configured?

Reply:
is_lapic_pt_configured is needed in arch/x86/guest/vmcs.c to switch to LAPIC Passthrough upon guest switch to use x2APIC from xAPIC.

Reply:
Use is_lapic_pt_enabled here